src/accessibility/describe.js

```diff
@@ -430,18 +430,15 @@ function _descElementLabelHTML(dummyDOM, cnvId, name, inner) {
         `<table id="${cnvId + labelTableId}"></table>`
       );
   }
-  //if it is the first time this element is being added to the table
-  if (dummyDOM.querySelector('#' + cnvId + fallbackTableId)) {
-    //create a table row for the element label description
-    let tableRow = document.createElement('tr');
-    tableRow.id = cnvId + labelTableElId + name;
-    dummyDOM.querySelector('#' + cnvId + labelTableId).appendChild(tableRow);
-    //update element label description
-    dummyDOM.querySelector(
-      '#' + cnvId + labelTableElId + name
-    ).innerHTML = inner;
-    return dummyDOM;
-  }
+  //create a table row for the element label description
+  let tableRow = document.createElement('tr');
+  tableRow.id = cnvId + labelTableElId + name;
+  dummyDOM.querySelector('#' + cnvId + labelTableId).appendChild(tableRow);
+  //update element label description
+  return [
+    dummyDOM.querySelector('#' + cnvId + labelTableElId + name),
```

Comment:
similar questions/comments here as for _describeFallbackHTML with this return clause and the insertion of the HTML earlier on